src/main/java/com/ss/ita/rozetka/pageobject/utils/PageUtil.java

```diff
@@ -19,4 +19,14 @@ public static boolean isElementVisible(String elementXPath) {
             return false;
         }
     }
+
+    public static boolean isElementInvisible(String elementXPath) {
```

Comment:
don't we have ElementUtil or some thing for things like that?

Reply:
It's PageUtil in this case, we haven't anouther class